.kres.yaml

```diff
@@ -113,4 +113,9 @@ spec:
 ---
 kind: auto.CI
 spec:
-    provider: drone
+  provider: drone
```

Comment:
this can already use github actions

Reply:
one step at a time, I need to move to GHA, but in a separate PR :)